libbpf-rs/src/program.rs

```diff
@@ -70,6 +70,11 @@ impl OpenProgram {
         };
         util::parse_ret(ret)
     }
+
+    pub fn set_prog_flags(&self, flags: u32) -> Result<()> {
```

Comment:
Should be `set_flags` ?

Reply:
There is a method `set_prog_type` so I was following that pattern. I can change it though.

Reply:
good point, shouldn't prog_type and set_prog_type be just `type` and `set_type` to match libbpf's API? Though I suspect "type" is reserved keyword in Rust?

Reply:
Yep, `type` is a keyword.